distributions/distribution-resources/src/main/resources/deb/systemd/openhab2.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
-Description=Starts and stops the openHAB Home Automation Bus
-Documentation=http://www.openhab.org
+Description=openHAB 2 Home Automation Bus Solution
+Documentation=http://docs.openhab.org
```

Comment:
"A space-separated list of URIs" - there should hence only be a single "Documentation=" entry here.
Btw, I prefer https://community.openhab.org/ (note the https), since the forum is SSL-enabled.